sdk/python/tests/test_historical_retrieval.py

```diff
@@ -441,6 +441,23 @@ def test_historical_features_from_bigquery_sources(
             ],
         )
 
+        # Just a dry run, should not create table
+        bq_dry_run = job_from_sql.to_bigquery(dry_run=True)
+        with pytest.raises(Exception):
```

Comment:
Do you have access to a more accurate exception like `TableNotFound` ?

Reply:
actually, since we won't return the path, can just test that nothing is returned when calling dry run :)

Reply:
changed it to reflect that